### DISCUSSION TITLE
inherited access on a private package registry

### DISCUSSION THREAD

**Discussion by jarblz** [community/community#36667] [Actions]
Hi there, I recently published a npm package through a private registry owned by my org, and I noticed that the "inherited access" checkbox on the package settings page was automatically unchecked.  

From reading the docs, I was under the impression that the access should be inherited from the repository by default.  Is there a specific setting somewhere that would prevent this? Or maybe an option i need to specify when publishing?  

Screenshot of the checkbox i'm talking about: 
![Package_pt-node-logger](https://user-images.githubusercontent.com/7277190/196718856-4badf8c7-cf27-4926-9eaf-d614d7a17665.png)

**Comment by ossanna16** [comment]
Hey there @jarblz and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by BenSeymourODB** [comment]
Having this same issue, though with NuGet packages instead of npm. Not sure how to specify that I want my packages to be connected to a particular repo and inherit permissions on the first push - so far I've had to set both of these manually each time I publish a package for the first time.
